Gemfile

```diff
@@ -19,7 +19,7 @@ gem 'dough-ruby',
     ref: 'cf08913'
 gem 'geocoder'
 gem 'kaminari'
-gem 'mas-rad_core', '0.0.85'
+gem 'mas-rad_core', '0.0.87'
```

Comment:
If possible can we separate this out into a "bump" commit? It's really useful to be able to see clearly in the git log when this has been done :)